scripts/ec2cleanup.py

```diff
@@ -218,8 +220,33 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    buckets = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name in bucket:
+                buckets.append(bucket)
+        deleted_buckets = provider_mgmt.delete_s3_buckets(bucket_names=buckets)
```

Comment:
Question:
what if `bucket_name` not matched with a single button means `buckets` list is empty? In this case, can we have to call `delete_s3_buckets` method form `werapanapi`?

Reply:
I have moved this call to delete_s3_buckets to next if block. Which handles bucket_name match thing.